pkg/operator/starter.go

```diff
@@ -336,13 +338,19 @@ func prepareOauthOperator(controllerContext *controllercmd.ControllerContext, op
 		controllerContext.EventRecorder,
 	)
 
+	systemCABundle, err := ioutil.ReadFile("/etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem")
+	if err != nil {
+		// this may fail route-health checks in proxy environments
+		klog.Warningf("Unable to read system CA from /etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem: %v", err)
```

Comment:
why not make this check during the route healthcheck and report it in message to user?

Reply:
it's not a check, you're loading the full file which might be quite long, you don't want this io operation appearing all the time

Reply:
i think a simple "file exists" check (os.Stat) would be cheap?

Reply:
heh, I see why this is confusing, I added a new argument but did not wire it :facepalm: 

The contents of the file are being used. I do not care whether it exists or not.